person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_accessor :id, :name, :age
```

Comment:
- [ ] Kindly consider id should only be able to be retrieved so only the Getter is defined `attr_reader :id`